py.test has lost the nice assertion failure display

when fixing #1930, I noticed that when the assertion fails, it just says "AssertionError", it does not display the values of the terms in the assertion expression.

E.g. like here:
```
__________________________________________________________ TestLocationWithoutEnv.test_file ___________________________________________________________

self = <borg.testsuite.helpers.TestLocationWithoutEnv object at 0x7fced6086940>

    def test_file(self):
        #monkeypatch.delenv('BORG_REPO', raising=False)
        assert repr(Location('xfile:///some/path::archive')) == \
>           "Location(proto='file', user=None, host=None, port=None, path='/some/path', archive='archive')"
E       AssertionError

borg/testsuite/helpers.py:48: AssertionError
```
The lhs value would have been interesting...

--- a/conftest.py
+++ b/conftest.py
@@ -4,6 +4,8 @@ import sys
 
 import pytest
 
+# needed to get pretty assertion failures in unit tests:
+pytest.register_assert_rewrite('borg')
 
 # This is a hack to fix path problems because "borg" (the package) is in the source root.
 # When importing the conftest an "import borg" can incorrectly import the borg from the

